subprojects/wrapper/src/main/java/org/gradle/wrapper/Download.java

```diff
@@ -75,6 +77,8 @@ private void downloadInternal(URI address, File destination)
             addBasicAuthentication(address, conn);
             final String userAgentValue = calculateUserAgent();
             conn.setRequestProperty("User-Agent", userAgentValue);
+            conn.setConnectTimeout(CONNECTION_TIMEOUT_SECONDS);
+            conn.setReadTimeout(READ_TIMEOUT_SECONDS);
```

Comment:
Both of these timeouts are in milliseconds, not seconds.

Reply:
Thanks, done.